autoreduce_frontend/reduction_viewer/views/run_summary.py

```diff
@@ -100,12 +99,16 @@ def run_summary_run(request, history, instrument_name=None, run_version=0, run_n
     page_type = request.GET.get('sort', '-run_number')
     next_run, previous_run, newest_run, oldest_run = get_navigation_runs(instrument_name, run, page_type)
 
+    script_present = ReductionScript(instrument_name).exists()
+    software_form = RerunForm(script_present=script_present)
```

Comment:
I know it is not in the scope of this PR, but why is this variable named so differently to the thing it is an instance of?